src/app/backend/resource/job/detail.go

```diff
@@ -91,6 +91,11 @@ func GetJobDetail(client k8sClient.Interface, metricClient metricapi.MetricClien
 
 func toJobDetail(job *batch.Job, eventList common.EventList, podList pod.PodList, podInfo common.PodInfo,
 	nonCriticalErrors []error) JobDetail {
+	// This pod info for jobs should be get from job status, similar to kubectl describe logic.
+	podInfo.Running = job.Status.Active
+	podInfo.Succeeded = job.Status.Succeeded
+	podInfo.Failed = job.Status.Failed
```

Comment:
It should be changed in `getJobPodInfo()` method from `resource/job/pods.go` file.